tests/call-centre/package.json

```diff
@@ -1,17 +1,14 @@
 {
   "name": "laa-cla-e2e-tests",
   "version": "1.0.0",
-  "description": "",
-  "directories": {
-    "test": "tests"
-  },
+  "description": "Runs the Civil Legal Aid call centre tests",
```

Comment:
_question_
I vaguely recall a conversation we had about naming these tests. I think it was about "call centre" or "case handling system". Should we choose now?

Reply:
I'll rename stuff to case handling system based on how it's been [described here](https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/785744082/Civil+Legal+Advice+CLA):

>The service is currently provided by Agilisys but hosted and maintained by MOJ (Ministry of Justice) Digital via Cloud-based platform Skyscape and consists of a call centre with ‘front line’ staff who take calls from users wishing to access the service and a **Case Handling System (CHS) where call operators and legal providers log and manage cases**.